android/src/main/java/com/zmxv/RNSound/RNSoundModule.java

```diff
@@ -1,18 +1,15 @@
 package com.zmxv.RNSound;
 
-import static android.os.Build.*;
-
 import android.app.Activity;
 import android.content.Context;
 import android.content.res.AssetFileDescriptor;
-import android.media.AudioDeviceInfo;
 import android.media.AudioManager;
 import android.media.AudioAttributes;
 import android.media.AudioFocusRequest;
 import android.media.MediaPlayer;
 import android.media.MediaPlayer.OnCompletionListener;
 import android.media.MediaPlayer.OnErrorListener;
-import android.os.Build.VERSION;
```

Comment:
Why did you change this import? If we want to merge a newer version or RNSound into our fork, changes like this add unnecessary complexity

Reply:
It used to be like `import android.os.Build;` before, I could not get it compiled with the change to `import android.os.Build.VERSION;` so changed it back.

Reply:
Clear